packages/react-components/react-accordion/src/components/Accordion/Accordion.types.ts

```diff
@@ -42,6 +42,7 @@ export type AccordionProps<Value = AccordionItemValue> = ComponentProps<Accordio
   multiple?: boolean;
 
   /**
+   * @deprecated Arrow keyboard navigation will be dropped.
```

Comment:
```suggestion
   * @deprecated Arrow keyboard navigation is not recommended for accordions. Consider using Tree if arrow navigation is a hard requirement.
```
just adding some more specific wording & recommendation

Reply:
Great, thanks 👍🏻